src/Authentication.Abstractions/AzureEnvironment.cs

```diff
@@ -29,11 +29,10 @@ namespace Microsoft.Azure.Commands.Common.Authentication.Abstractions
     /// location fo service-specific endpoints, and information for bootstrapping authentication
     /// </summary>
     [Serializable]
-    public class AzureEnvironment : IAzureEnvironment, IEquatable<AzureEnvironment>
+    public partial class AzureEnvironment : IAzureEnvironment, IEquatable<AzureEnvironment>
     {
         private const string ArmMetadataEnvVariable = "ARM_CLOUD_METADATA_URL";
 
-        private const string DefaultArmMetaDataEndpoint = "http://discover.azure.com/";
```

Comment:
The default point is no longer accessible since years ago. However our product spends time to try to access it every-time we import Az.Accounts and so just remove it.

Reply:
We'll keep the default endpoint because we still need it to query the endpoints when `Add-AzEnvironment -AutoDiscover`